src/core/xmlstreamwriter.cc

```diff
@@ -19,75 +19,28 @@
 
 #include "src/core/xmlstreamwriter.h"
 
-#include <QtCore/QFile>
-#include <QtCore/QString>
-#include <QtCore/QTextCodec>
-#include <QtCore/QXmlStreamWriter>
+#include <QtCore/QString>           // for QString
+#include <QtCore/QXmlStreamWriter>  // for QXmlStreamWriter
+#include <QtCore/QtGlobal>          // for QT_VERSION, QT_VERSION_CHECK
 
 // As this code began in C, we have several hundred places that write
```

Comment:
This comment has gotten orphaned from its code. Please consider shoving it closer to the body of writeOptionalTextElement.